### ISSUE TITLE
`create_msd_json_from_bids.py` - missing `ses-` tag for labels

### ISSUE THREAD

**Issue by valosekj** [ivadomed/templates#6]
The [`create_msd_json_from_bids.py`](https://github.com/ivadomed/templates/blob/main/dataset_conversion/create_msd_json_from_bids.py) script does not include the `ses-` tag for the label files in the output `dataset.json` file.

For example, notice that `ses-01` is missing in the path for the label file:

```
{
    "image": "<DATA>/sci-zurich/sub-zh64/ses-01/anat/sub-zh64_ses-01_acq-sag_T2w.nii.gz",
    "label": "<DATA>/sci-zurich/derivatives/labels/sub-zh64/anat/sub-zh64_ses-01_acq-sag_T2w_lesion-manual.nii.gz"
},
```

We are also missing the `ses-` tag in our examples, for instance, here:
https://github.com/ivadomed/templates/blob/d4c8f7e9fdb5a205d0c79f52fcef8da89abe26a4/dataset_conversion/create_msd_json_from_bids.py#L32-L33

**Comment by naga-karthik** [comment]
Thanks for pointing it out! This has been fixed in commit 1d9b9dea2f8efbb02cd7a2849a77ba7fe097d4a2

**Comment by valosekj** [comment]
Thanks for the quick fix! Tested and now working as expected!
